styles.css

```diff
@@ -0,0 +1,328 @@
+
+/*//////////////////////////
+Table of Contents
+ 1. Typografie
+ 2. Global styles
+ 3. Layout
+ 3.1 Reusables
+ 3.2 Others
+ 4. Areas
+ 4.1 Header
+ 4.2 Main
+ 4.3 Footer
+ 5. Media Queries
+ ///////////////////////////*/
+
+/*/////////////////////////////////////////
+        1.Typografie
+////////////////////////////////////////// */
+
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@700&family=Roboto&display=swap');
+
+
+/*/////////////////////////////////////////
+        2.Global Styles
+////////////////////////////////////////// */
+/* Hierin wordt opgenomen: CSS reset en
+ Variabelen */
+
+/*CSS reset*/
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding:0;
+}
+/*variabelen*/
+
+:root {
+    --primary-color:#5722CD;
+    --primary-color-light:  #4845E4;
+    --default-radius: 3px;
+
+}
+
+/*/////////////////////////////////////////
+        3.Layout
+////////////////////////////////////////// */
+
+.intro__outer-container,
+.header__outer-container,
+.projects__outer-container,
+.services__outer-container,
+.holder__outer-container{
+    display:flex;
+    justify-content: center;
+    width:100%;
+  /*  border: 3px solid red;*/
+}
+.header__inner-container,
+.intro__inner-container,
+.projects__inner-container,
+.services__inner-container,
+.holder__inner-container {
+    flex-basis: 1400px;
+    margin: 0 20px;
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+}
+.header__outer-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    background: var(--primary-color);
+    background: linear-gradient(300deg, var(--primary-color) 100%);
+    /*border: 3px solid red;*/
+}
+.header__inner-container {
+    /*border: 3px solid green;*/
+    flex-direction: column;
+    align-items: center;
+    background-color:#5722CD;
+    --primary-color-light:  #4845E4;
+}
+.header__navbar {
+    width: 100%;
+}
+.header__navbar-link {
+    color: var (--text-color-light);
+    text-transform: capitalize;
+    text-decoration: underline 3px rgba(0.0,0,0,);
+}
+.header__navbar-list {
```

Comment:
Hier hoef je niet nog apart een classname te declareren: je had er hier voor kunnen kiezen om een descendant selector te gebruiken. 
Bijvoorbeeld: `header__navbar ul { ... }`

Dit zorgt er ook voor dat je HTML en CSS wat makkelijker leesbaar is :)